samples/azure-quantum/chemistry/QuantumPhaseEstimation/GetEnergyQPE.qs

```diff
@@ -0,0 +1,43 @@
+namespace Microsoft.Quantum.Chemistry.QPE {
+    open Microsoft.Quantum.Core;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Chemistry;
+    open Microsoft.Quantum.Chemistry.JordanWigner;  
+    open Microsoft.Quantum.Simulation;
+    open Microsoft.Quantum.Characterization;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+
+    /// # Summary
+    /// Get the molecule energy using Quantum Phase Estimation.
+    ///
+    /// # Input
+    /// ## JWEncodedData
+    /// Jordan-Wigner encoded data.
+    /// ## nBitsPrecision
+    /// Number of bits of precision.
+    /// ## trotterStepSize
+    /// Trotter step size.
+    /// ## trotterOrder
+    /// The Trotter order to use.
+    ///
+    /// # Output
+    /// Tuple of estimated phase and energy.
+    operation GetEnergyQPE (
```

Comment:
Similarly here, I think "estimate" would be a more accurate verb to use; we in general don't have direct access to the energy, such that there is always finite uncertainty in our estimates.

Reply:
I'd like to keep the names of the operations the same as in the notebook we used for the //build demo, if that's okay with you. I think it's clear from the context that we're estimating the energy, not getting the accurate value.